tests/devices/default_qubit/test_default_qubit_tracking.py

```diff
@@ -251,3 +250,39 @@ def test_single_expval(mps, expected_exec, expected_shots):
         assert dev.tracker.totals["executions"] == 3 * expected_exec
         assert dev.tracker.totals["simulations"] == 1
         assert dev.tracker.totals["shots"] == 3 * expected_shots
+
+
+@pytest.mark.xfail  # TODO Prod instances are not automatically
+def test_multiple_expval_with_prods():
+    mps, expected_exec, expected_shots = (
+        [qml.expval(qml.PauliX(0)), qml.expval(qml.PauliX(0) @ qml.PauliY(1))],
+        1,
+        10,
+    )
+    dev = qml.device("default.qubit")
+    tape = qml.tape.QuantumScript([], mps, shots=10)
+
+    with dev.tracker:
+        dev.execute(tape)
+
+    assert dev.tracker.totals["executions"] == expected_exec
+    assert dev.tracker.totals["simulations"] == 1
+    assert dev.tracker.totals["shots"] == expected_shots
+
+
+@pytest.mark.usefixtures("use_legacy_opmath")
+def test_multiple_expval_with_Tensors_legacy_opmath():
+    mps, expected_exec, expected_shots = (
+        [qml.expval(qml.PauliX(0)), qml.expval(qml.operation.Tensor(qml.PauliX(0), qml.PauliY(1)))],
```

Comment:
Is this necessary? `Tensor` is automatically created when using `@` in a legacy opmath context right?

Reply:
No not necessary, just for good measure. Is it a problem?

Reply:
No, I'm just trying to figure out what the recommended approach is.